src/SmartComponents/Portfolio/PortfolioItems.js

```diff
@@ -1,32 +1,33 @@
-import React, { Component } from 'react';
-import { withRouter } from 'react-router-dom';
+import React, { Component, cloneElement } from 'react';
+import { bindActionCreators } from 'redux';
+import { withRouter, Link, Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 import propTypes from 'prop-types';
 import { parse } from 'querystring';
 import { Section } from '@red-hat-insights/insights-frontend-components';
-import { Toolbar, ToolbarGroup, ToolbarItem, ToolbarSection } from '@patternfly/react-core';
+import { Button, Toolbar, ToolbarGroup, ToolbarItem, ToolbarSection } from '@patternfly/react-core';
 import ContentGallery from '../../SmartComponents/ContentGallery/ContentGallery';
 import { fetchPortfolioItems, fetchPortfolioItemsWithPortfolio } from '../../Store/Actions/PortfolioActions';
 import { consoleLog } from '../../Helpers/Shared/Helper';
 import MainModal from '../Common/MainModal';
-import './portfolioitems.scss';
+import RemovePortfolioItems from '../../SmartComponents/Portfolio/RemovePortfolioItems';
 
 class PortfolioItems extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            showItems: '',
-            filteredItems: []
-        };
-        consoleLog('PortfolioItems props: ', props);
-    }
+    state = {
+      showItems: '',
+      filteredItems: [],
+      selectItems: [],
+      isLoaded: false,
+    };
 
     fetchData(apiProps) {
         if (apiProps && apiProps.portfolio) {
-            this.props.fetchPortfolioItemsWithPortfolio(apiProps.portfolio);
+            this.props.fetchPortfolioItemsWithPortfolio(apiProps.portfolio).then(() =>
```

Comment:
You can just call this.props.fetchPortfolioItemsWithPortfolio (apiProps.portfolio);  here 
The middleware will dispatch the ActionTypes.FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO + '_FULFILLED' action and set the payload to the api result and isLoading to false.